### PR TITLE
docs(docx): Adding Hooks to new docs site [JOB-111817]

### PR DESCRIPTION
**Pull request by Beccaneer**
Screenshot 2024-12-17 at 12 45 56 PM
Screenshot 2024-12-17 at 12 52 55 PM

### Added

- Added Hooks pages to the new docs site
- Added Hooks pages to sidenav
- Added Hooks collection page

### Changed

- In order to get examples to display properly, all usages of `<Story>` in the hook storybook stories files needed to be removed. The Story's inner contents have been moved to a separate component file and then imported to get around this.

## Testing

On the new docs site:
- ensure hooks are in left side nav and links work
- ensure you can get to the hooks collection page and links work
- check some hooks pages and make sure things display as expected

On the old docs site:
- make sure hooks pages load as expected

Changes can be
[tested via Pre-release](https://github.com/GetJobber/atlantis/actions/workflows/trigger-qa-build.yml)

---

[In Atlantis we use Github's built in pull request reviews](https://help.github.com/en/articles/about-pull-request-reviews).

Random photo of Atlantis

### CODE REVIEW

File: docs/hooks/useInView.stories.mdx [hunk 1]
```diff
@@ -14,32 +12,5 @@ import { useInView } from "@jobber/hooks/useInView";
...
-            </Content>
-          </div>
-        </Content>
-      );
-    }}
-  </Story>
+  <UseInView />
 </Canvas>
-
-<ArgsTable of={hooks.useInView} />
```
Comment (Beccaneer) [inline docs/hooks/useInView.stories.mdx:45]: This didn't seem to be working anywhere so I removed it

**Review by taylorvnoj — CHANGES_REQUESTED** [review]
Looking good! A couple of nits that can probably turn into tickets.
But blocking for me, is adding Hooks to Search and Home page please 😄

File: packages/site/src/maps/hooksContent.tsx [hunk 2]
```diff
@@ -0,0 +1,100 @@
+import UseAssertDocs from "@atlantis/docs/hooks/useAssert.stories.mdx";
```
[the remark this answers is not in the document]
Comment (Beccaneer) [reply packages/site/src/maps/hooksContent.tsx:1]: Yeah I realized that, but also wasn't sure what Chris wanted. I'll make a ticket :)

[the remark this answers is not in the document]
Comment (Beccaneer) [reply packages/site/src/maps/hooksContent.tsx:1]: https://jobber.atlassian.net/browse/JOB-112176

File: docs/hooks/UseResizeObserver.tsx [hunk 3]
```diff
@@ -0,0 +1,33 @@
+import React from "react";
```
[the remark this answers is not in the document]
Comment (Beccaneer) [reply docs/hooks/UseResizeObserver.tsx:1]: https://jobber.atlassian.net/browse/JOB-112177

**Review by taylorvnoj — APPROVED** [review]
LGTM thanks for the tickets and the changes!

**Comment by taylorvnoj** [comment]
Oops - didn't notice the CI failure @Beccaneer

**Pull request merged**